libs/json-api/json-api-nestjs/src/lib/helper/orm/methods/get-all/get-all.ts

```diff
@@ -250,7 +249,6 @@ export async function getAll<E extends Entity>(
     );
   }
   const resultData = await resultQuery.getMany();
-  console.log(resultData);
```

Comment:
If you need this during development why not to use nestjs building logger function in the debug only? Then we can activate or deactivate the debug log when necessary. 
https://docs.nestjs.com/techniques/logger

Reply:
No, it was my mistake.